connect/transforms/src/main/java/org/apache/kafka/connect/transforms/ReplaceField.java

```diff
@@ -38,6 +40,7 @@
 import static org.apache.kafka.connect.transforms.util.Requirements.requireStruct;
 
 public abstract class ReplaceField<R extends ConnectRecord<R>> implements Transformation<R> {
+    private static final Logger log = LoggerFactory.getLogger(ReplaceField.class);
```

Comment:
I'm not sure any of these log messages add value since TransformationChain has log messages with the records, which should be sufficient to know whether the schemas are null.